instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/RatpackTelemetryBuilder.java

```diff
@@ -29,7 +29,7 @@
 /** A builder for {@link RatpackTelemetry}. */
 public final class RatpackTelemetryBuilder {
 
-  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.ratpack-1.4";
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.ratpack-1.7";
```

Comment:
Is there a test now that fails when these are wrong?

Reply:
No, there isn't, unfortunately -- I went over all usages of `Instrumenter#builder()` and fixed them all manually. Currently I don't have any good idea about how to verify this in a smart way that does not require adding assertions to every test.